cms/plugins/appentities/company/http/controllers/CompanyController.php

```diff
@@ -0,0 +1,61 @@
+<?php namespace AppEntities\Company\Http\Controllers;
+
+use Illuminate\Routing\Controller;
+use Appentities\Company\Models\Company;
+use AppEntities\Company\Http\Resources\CompanyResource;
+
+class CompanyController extends Controller
+{
+
+    const PER_PAGE = 10;
+
+    public function index() {
+
+        $searchQuery = input('search_query');
+        $revenueMin = input('revenue_min');
+        $revenueMax = input('revenue_max');
+        $profitsMin = input('profits_min');
+        $profitsMax = input('profits_max');
+        $order  = input('order') ?? 'revenue';
```

Comment:
mam pocit ze sa da pouzit rdsej toto
```suggestion
        $order  = input('order', 'revenue');
```